docs/docs/guides/evaluation/guardrails_and_monitors.md

```diff
@@ -5,139 +5,482 @@ import TabItem from '@theme/TabItem';
+
+:::tip
+Every scorer result is automatically stored in Weave's database. This means your guardrails double as monitors without any extra work! You can always analyze historical scorer results, regardless of how they were originally used.
+:::
+
+### Using the `.call()` Method
+
+To use scorers with Weave ops, you'll need access to both the operation's result and its tracking information. The `.call()` method provides both:
 
 ```python
-class MyScorer(Scorer):
-    def score(self, output: str) -> float:
-        """
-        Evaluate the given result and return a score between 0 and 1.
-        
-        Args:
-            result: The LLM-generated content to evaluate
-            
-        Returns:
-            float: Score indicating quality/safety (0 = fail, 1 = pass)
-        """
-        return 0.8  # Example score
+# Instead of calling the op directly:
+result = generate_text(input)  # Primary way to call the op but doesn't give access to the Call object
+
+# Use the .call() method to get both result and Call object:
+result, call = generate_text.call(input)  # Now you can use the call object with scorers
 ```
 
-### Applying Scorers
+:::tip Why Use `.call()`?
+The Call object is essential for associating the score with the call in the database. While you can directly call the scoring function, this would not be associated with the call, and therefore not searchable, filterable, or exportable for later analysis.
+
+For more details about Call objects, see our [Calls guide section on Call objects](../tracking/tracing.mdx#getting-a-handle-to-the-call-object-during-execution).
+:::
+
+## Getting Started with Scorers
+
+### Basic Example
 
-Scorers are applied to operations using the `apply_scorer` method, which returns an `ApplyScorerResult`:
+Here's a simple example showing how to use `.call()` with a scorer:
 
 ```python
-@dataclass
-class ApplyScorerSuccess:
-    result: Any  # The original operation result
-    score_call: Call  # The scoring operation call object
+import weave
+from weave import Scorer
+
+class LengthScorer(Scorer):
+    @weave.op
+    def score(self, output: str) -> dict:
+        """A simple scorer that checks output length."""
+        return {
+            "length": len(output),
+            "is_short": len(output) < 100
+        }
+
+@weave.op()
+def generate_text(prompt: str) -> str:
+    return "Hello, world!"
+
+# Get both result and Call object
+result, call = generate_text.call("Say hello")
+
+# Now you can apply scorers
+await call.apply_scorer(LengthScorer())
 ```
 
-Basic scorer application:
+## Using Scorers as Guardrails
 
-```python
-# Get both operation result and Call object
-result, call = op.call(user_input)
+Guardrails act as safety checks that run before allowing LLM output to reach users. Here's a practical example:
 
-# Apply scorer and get evaluation results
-evaluation = await call.apply_scorer(scorer)
+```python
+import weave
+from weave import Scorer
+
+@weave.op()
```

Comment:
Should be consistent with parenthesis in `.op()` or `.op`, previous example has no parenthesis

Reply:
got it